server/svix-server/src/v1/endpoints/auth.rs

```diff
@@ -16,11 +17,24 @@ pub struct DashboardAccessOut {
     pub token: String,
 }
 
+#[derive(Deserialize, Serialize, Validate, JsonSchema)]
+pub struct DashboardAccessIn {
+    #[validate]
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Can we change this to just be a `EventTypeFeatureFlagSet` (or `HashSet<EventTypeFeatureFlag>` if you go that route - basically just drop the `Option`). I think Serde should be able to still default it to an empty set if nothing is passed in.

Reply:
Done.